src/main/java/calculator/parser/InputParser.java

```diff
@@ -0,0 +1,21 @@
+package calculator.parser;
+
+public class InputParser {
+    public static final String SPLIT_SPACE_CHAR = " ";
```

Comment:
```suggestion
    private static final String SPLIT_SPACE_CHAR = " ";
```
클래스 내부에서만 사용되니 private으로 감추는게 좋겠네요